README.md

```diff
@@ -1 +1,216 @@
-# Bookstore
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. 
+  <img src="./images/moon.png" alt="logo" width="140"  height="auto" /> -->
```

Comment:
- [ OPTIONAL ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1: